migrate-configuration-v2.rst

```diff
@@ -0,0 +1,90 @@
+.. post:: May 31, 2023
+   :tags: builders, deprecation
+   :author: Manuel
+   :location: BCN
+   :category: Changelog
+
+Migrate your project to ``.readthedocs.yaml`` configuration file v2
+===================================================================
+
+We are announcing the **deprecation of building without a configuration file** (``.readtehdocs.yaml``)
+together with the deprecation of configuration file v1 as well.
+
+Read the Docs *will soon start requiring* a ``.readthedocs.yaml`` configuration file
+for all the projects to build their documentation successfully.
+We will stop creating a default configuration file *behind the scenes*, making assumptions for your projects
+and trying our best to make the builds succeed.
+
+We plan to deploy these changes and **start failing builds without a configuration file on September 2023**.
+This matches the period when ``urllib3`` `drops support for OpenSSL 1.1.1 <https://github.com/urllib3/urllib3/issues/2168>`_
+which is OpenSSL version that Ubuntu 18.04 has installed and it's used when the project doesn't have a configuration file.
+
+The main reason behind this decision is it's pretty hard to maintain a set of defaults that are useful
+for the amount of users we have and the number of different tools they use to build documentation.
+Even when the defaults we chose were useful for a period of time,
+we found they were almost impossible to update to newer versions of dependencies
+because doing so would break a lot of projects that weren't using a configuration file
+and pinning their dependencies (e.g. OS, Python version, dependencies, etc).
```

Comment:
```suggestion
*Behind the scenes*, we used to assume a default configuration file for projects that didn't have one. However, a "one size fits all" default project setup isn't sustainable anymore, since it requires us to find a common ground for projects created in a period of over 10 years!

We would also much rather encourage all the positive aspects of having a configuration file. Maintaining your Read the Docs configuration together with your documentation allows you to version them together.

Moving forward, this change will make our platform much less complex. It means that we can introduce new features faster and that users will experience fewer issues due to the unstable nature of these one-size-fits-all legacy default setups.
```

I suggest to turn this into a "Background" section and move it to the end of the blog post. That way, they deprecation message and the practical parts can get the focus.